doc/examples/bulk.rst

```diff
@@ -141,11 +141,12 @@ and fourth operations succeed.
    'upserted': [],
    'writeConcernErrors': [],
    'writeErrors': [{'code': 11000,
-                    'errmsg': '...E11000...duplicate key error...',
+                    'errmsg': 'E11000 duplicate key error collection: '
+                              'bulk_example.test index: _id_ dup key: { _id: 1 }',
                     'index': 0,...
                     'op': {'_id': 1}},
                    {'code': 11000,
-                    'errmsg': '...E11000...duplicate key error...',
+                    'errmsg': '',
```

Comment:
The server is returning an empty errmsg seems unexpected and likely a bug in v6.0-latest. Could you file a server ticket for this? 

Perhaps for now we can just do `'...'` for the second errmsg field?